sites/public/pages/listings.tsx

```diff
@@ -181,6 +184,9 @@ const ListingsPage = () => {
   // eslint-disable-next-line @typescript-eslint/unbound-method
   const { handleSubmit, register, errors } = useForm()
   const onSubmit = (data: ListingFilterState) => {
+    if (data?.[FrontendListingFilterStateKeys.includeNulls] === false) {
+      delete data?.[FrontendListingFilterStateKeys.includeNulls]
```

Comment:
super nit: you don't need the `?` in this line, since it's guaranteed that the field exists if we get here.

Reply:
I get a parsing error when I remove the `?`.
`Parsing error: Identifier expected.eslint
(parameter) data: ListingFilterState`